docs/content/gameobject/readers-writers.md

```diff
@@ -58,8 +58,9 @@ The exact type depends on the component that the reader is generated for.
 
 | Field         	| Type               	| Description                	|
 |-------------------|------------------------|--------------------------------|
-| Data  	| TSpatialComponentData              	| The data stored inside the component that this Reader is associated with. |
-| Authority | [Authority]({{urlRoot}}/content/glossary#schema) | The authority state of the current worker of the component that this Reader is associated with. |
+| EntityId | `EntityId`                 | The EntityId of the underlying SpatialOS entity of this Reader. |
```

Comment:
```suggestion
| EntityId | `EntityId`                 | The entity ID of the underlying SpatialOS entity of this Reader. |

```

As we use "entity ID" in related pages, we should be consistent, I think. (Might have missed this in a first pass.)